getting_started_materials/intro_tutorials_and_guides/02_Introduction_to_cuDF.ipynb

```diff
@@ -42,9 +42,35 @@
   },
   {
    "cell_type": "code",
```

Comment:
@taureandyernv do we want to have the output of the cells in the notebooks?

Reply:
IMO, yes we do. Many of these notebooks are points of reference or quick "how tos" where people look at them on GitHub without actually cloning or having to otherwise run them to see what the output should be.

I don't have a data to back it up, but I believe this is probably the most common usage for these notebooks. We certainly link community folks to them often, and seeing the output helps understand what's happening quickly.

Reply:
I agree we need certain amount of output in the notebooks. However, there are some cells where we just print the dataset or large amounts of information. I think it would be good if we keep the cells that help the users to understand and highlight the notebooks and remove the excessive prints.

Reply:
yes, let's keep the outputs